core/src/main/kotlin/riven/core/service/catalog/ManifestLoaderService.kt

```
@@ -0,0 +1,122 @@
+        val modelIndex = mutableMapOf<String, JsonNode>()
+        var modelsLoaded = 0
+
+        for (scanned in scannedModels) {
+            try {
+                val resolved = resolverService.resolveManifest(scanned, emptyMap())
+                upsertService.upsertManifest(resolved)
+                if (!resolved.stale) {
+                    modelIndex[scanned.key] = scanned.json
+                    modelsLoaded++
+                }
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)